ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Ledger/Tables.hs

```diff
@@ -13,13 +15,23 @@
 
 -- | See @'LedgerTables'@
 module Ouroboros.Consensus.Ledger.Tables (
-    -- * Kinds
-    LedgerStateKind
-  , MapKind
+    -- * Re-exports
+    module Ouroboros.Consensus.Ledger.Tables.Combinators
+  , module Ouroboros.Consensus.Ledger.Tables.Common
     -- * Basic LedgerState classes
   , CanStowLedgerTables (..)
   , HasLedgerTables (..)
-  , HasTickedLedgerTables (..)
+  , HasTickedLedgerTables
+    -- * TODO: To be removed
```

Comment:
What do we need to remove these?

Reply:
Not much, but it mostly inflates the diff because identifiers have to be replaced. I can make a follow-up PR to remove these